scripts/prettier.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+const path = require('path');
+const chalk = require('chalk');
+const glob = require('glob');
+const runCommand = require('./_runCommand');
+
+const shouldWrite = process.argv[2] === 'write';
+const isWindows = process.platform === 'win32';
+const prettier = isWindows ? 'prettier.cmd' : 'prettier';
+const prettierCmd = path.resolve(__dirname, '../node_modules/.bin/' + prettier);
+
+const config = {
+  ignore: ['**/node_modules/**'],
+  options: {
+    'bracket-spacing': 'false',
+    'print-width': 120,
```

Comment:
What happens when we change this to 80 (or 100)? Prettiers output actually looks worse the longer the line-length is because it tries to cram everything into one line.